ext/twig/php_twig.h

```diff
@@ -2,34 +2,54 @@
    +----------------------------------------------------------------------+
    | Twig Extension                                                       |
    +----------------------------------------------------------------------+
-   | Copyright (c) 2011 Derick Rethans                                    |
+   | Copyright (c) 2011 Sensiolabs                                        |
    +----------------------------------------------------------------------+
    | Redistribution and use in source and binary forms, with or without   |
    | modification, are permitted provided that the conditions mentioned   |
    | in the accompanying LICENSE file are met (BSD-3-Clause).             |
    +----------------------------------------------------------------------+
    | Author: Derick Rethans <derick@derickrethans.nl>                     |
+   |         Julien PAULI <jpauli@php.net>                                |
    +----------------------------------------------------------------------+
  */
 
 #ifndef PHP_TWIG_H
 #define PHP_TWIG_H
 
-#define PHP_TWIG_VERSION "2.0.0-DEV"
+#define PHP_TWIG_VERSION "2.0.0"
+
+#define TWIG_TEMPLATE_CLASS_NAME      "Twig_Template"
+#define TWIG_ERROR_RUNTIME_CLASS_NAME "Twig_Error_Runtime"
 
 #include "php.h"
 
 extern zend_module_entry twig_module_entry;
-#define phpext_twig_ptr &twig_module_entry
-#ifndef PHP_WIN32
+zend_module_entry *get_module(void);
+
+#ifdef COMPILE_DL_TWIG
 zend_module_entry *get_module(void);
```

Comment:
is it expected to duplicate the line available just above in case ``COMPILE_DL_TWIG`` is defined ?